lib/eblox/data/post.ex

```diff
@@ -19,6 +19,40 @@ defmodule Eblox.Data.PostFSM do
 
   use Finitomata, @fsm
 
+  defmodule Tag do
+    @moduledoc false
+    @behaviour Md.Transforms
+
+    @href "/tags/"
+
+    @impl Md.Transforms
+    def apply(md, text) do
+      {:a, %{class: "tag", href: @href <> text}, [md <> text]}
```

Comment:
Unless we plan to format tags somewhere else the result `href` here needs to be URL encoded.

Reply:
Damn, of course, thanks.